Resources/Textures/_Harmony/Interface/Uplink/Serum.rsi/meta.json

```
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "size": {
+    "x": 64,
+    "y": 64
+  },
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/tgstation/tgstation/commit/53d1f1477d22a11a99c6c6924977cd431075761b , and modified by BoskiYourk for use in Harmony",
+  "states": [
+    {
+      "name": "gorilla_rampage"
+    }
+  ]
+}
```

Indentation should be 4 spaces wide in RSIs
```suggestion
{
    "version": 1,
    "size": {
        "x": 64,
        "y": 64
    },
    "license": "CC-BY-SA-3.0",
    "copyright": "Taken from https://github.com/tgstation/tgstation/commit/53d1f1477d22a11a99c6c6924977cd431075761b , and modified by BoskiYourk for use in Harmony",
    "states": [
        {
            "name": "gorilla_rampage"
        }
    ]
}
```